corehq/apps/locations/templates/locations/manage/location.html

```diff
@@ -83,6 +83,25 @@
       </script>
 
       {% if not form.is_new_location %}
+
+        <p class="lead">{{ location.name }} <small>{{ location.location_type_name }}</small></p>
+
+        <div class="btn-toolbar" style="margin-bottom: 20px;">
+
+          {% if location.get_id and location.location_type_object.can_have_children %}
+          <a class="btn btn-default" href="{% url "create_location" domain %}?parent={{ location.get_id }}">
```

Comment:
Think `btn-default` is appropriate for both of these?  They're not the primary action on the page.